az-reportal/src/main/java/azkaban/viewer/reportal/ReportalServlet.java

```diff
@@ -398,14 +398,14 @@ private void handleViewReportal(final HttpServletRequest req,
         }
         // View single log
         if (hasParam(req, "log")) {
-          page.add("view-log", true);
+          page.add("viewLog", true);
```

Comment:
Clarification: I am curious to know if this change and a few other changes for view-logs, view-preview, view-executions are mandated by the library or we are doing this to keep it clean. Thanks a lot for picking up this change.

Reply:
This is mandated by the library. Hyphens are no longer allowed. I believe underscores are allowed (_) if we would like to change from view-log to view_log.

Reply:
Thanks a lot @amielkelinkedin ! :)